src/components/DropdownMenu/README.md

```diff
@@ -4,42 +4,48 @@ A dropdown menu with a customizable toggle. By default, the toggle is an ellipsi
-| `switcherWrapperClassName` | `string`                                     |          |               | A value for the `className` prop of the `switcher`'s parent component.                              |
-| `defaultSwitcherProps`     | `ButtonProps`                                |          |               | Default `switcher` props.                                                                           |
-| `defaultSwitcherClassName` | `string`                                     |          |               | A value for the `className` prop of the default `switcher`.                                         |
-| `menuProps`                | `MenuProps`                                  |          |               | Overrides the default dropdown menu popup props.                                                    |
-| `onMenuToggle`             | `() => void`                                 |          |               | Called when the menu is opened or closed.                                                           |
-| `onSwitcherClick`          | `React.MouseEventHandler<HTMLElement>`       |          |               | Called when `switcher` is clicked.                                                                  |
-| `hideOnScroll`             | `boolean`                                    |          | `true`        | Specifies whether to hide the menu when a parent element is scrolled.                               |
-| `popupPlacement`           | `PopupPlacement`                             |          |               | Allowed positions of the menu popup.                                                                |
-| `popupClassName`           | `string`                                     |          |               | A value of the `className` prop of the menu popup.                                                  |
-| `children`                 | `ReactNode`                                  |          |               | Custom content inside the menu popup.                                                               |
+| Name                       | Type                                   |        Required        | Default value | Description                                                                                         |
+| :------------------------- | :------------------------------------- | :--------------------: | :------------ | :-------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| `items`                    | `(DropdownMenuItem \                   | DropdownMenuItem[])[]` |               | `[]`                                                                                                | Array of items. Nested arrays of items represent visually separated groups. |
```

Comment:
> <pre>\                   |</pre>

the spaces after the slash should break the escaping, which should result in a messed up table

there's at least another one below in the `size` prop row

Reply:
the IDE breaks that